include/boost/geometry/algorithms/detail/envelope/linestring.hpp

```diff
@@ -108,7 +116,9 @@ struct envelope_linear_on_spheroid
             Box segment_mbr;
             dispatch::envelope
                 <
-                    typename std::iterator_traits<iterator_type>::value_type
+                    typename std::iterator_traits<iterator_type>::value_type,
+                    0,
+                    DimensionCount
```

Comment:
In this case which dimensions exactly are handled in this `envelope` call? `0` and `1` or `0` to `DimensionCount`?